code/gradle.properties

```diff
@@ -21,14 +21,14 @@ android.useAndroidX=true
 moduleProjectName=edge
 moduleName=edge
 moduleAARName=edge-phone-release.aar
-moduleVersion=2.1.0
+moduleVersion=2.2.0
 
 mavenRepoName=AdobeMobileEdgeSdk
 mavenRepoDescription=Adobe Experience Platform Edge extension for the Adobe Experience Platform Mobile SDK
 mavenUploadDryRunFlag=false
 
 # production versions for production build
-mavenCoreVersion=2.0.0
+mavenCoreVersion=2.2.0
```

Comment:
Core version with parentId changes is also 2.2.0?

Reply:
Yes, https://github.com/adobe/aepsdk-core-android/releases/tag/2.2.0-core